### ISSUE TITLE
importlib.abc.Traversable.name does not match

### ISSUE THREAD

**Issue by 4251eaa2-4c23-4b4f-9153-295ef15b0d58** [python/cpython#89896]
BPO | [45733](https://bugs.python.org/issue45733)
--- | :---

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2021-11-06.01:23:27.490>
labels = ['type-bug', '3.9', '3.10', '3.11']
title = 'importlib.abc.Traversable.name does not match'
updated_at = <Date 2021-11-06.01:23:27.490>
user = 'https://bugs.python.org/jmg'
```

bugs.python.org fields:
```python
activity = <Date 2021-11-06.01:23:27.490>
actor = 'jmg'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = []
creation = <Date 2021-11-06.01:23:27.490>
creator = 'jmg'
dependencies = []
files = []
hgrepos = []
issue_num = 45733
keywords = []
message_count = 1.0
messages = ['405837']
nosy_count = 1.0
nosy_names = ['jmg']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue45733'
versions = ['Python 3.9', 'Python 3.10', 'Python 3.11']
```

</p></details>

**Comment by warsaw** [comment]
Yikes!  I just ran into this too.  I'll test it on newer versions and submit a PR.  Should be an easy documentation fix.

**Comment by warsaw** [comment]
Well, not so easy apparently because of the `importlib` docs got reorg'd in `main`, so it'll take a couple of PRs most likely.

**Comment by warsaw** [comment]
Turns out, this is fixed in `main` but there's another problem (#93610) for which I created a PR first.

**Comment by jaraco** [comment]
Looks like this work was done. Thanks warsaw and jmg. Please comment or re-open if the issue remains.
